enhancements/console/dynamic-plugins.md

```diff
@@ -272,6 +274,21 @@ available and the user must refresh their browser to see the changes. (We won't
 refresh the page automatically to avoid possibly losing data if the user is
 entering something into a form or the YAML editor.)
 
+### Localization
+
+Dynamic plugins will need to contain their own localization files and serve them
+per requested. All dynamic plugins must use a single localization namespace
+per language, named after the namespace, e.g. for `kubevirt-plugin` the filename
+would be `kubevirt-plugin.json`. Localization files need to be served under
+`/locales/{language}/{namespace}.json` path.
+All dynamic plugins are expected to use `plugin__` namespace prefix, e.g.
+`plugin__knative-plugin` or `plugin__kubevirt-plugin`. The request for the dynamic
```

Comment:
Maybe use `plugin__knative` and `plugin__kubevirt` as examples? I think in most cases you wouldn't have `-plugin` in the plugin name.

A code snippet showing a real example would be helpful.

```ts
const VMHeading = () => {
  const { t } = useTranslation();
  return <h1>{t('plugin__kubevrit~Virtual Machine')}</h1>;
};
```

Reply:
@spadgett hmmm, but wont the locale file contain the `-plugin` string? I though it would. Or are we counting that it will contain it and in that case we done need to `ns` to contain it ?

Reply:
I'm saying usually the plugin name will be something like `kubevrit` instead of `kubevirt-plugin`. Adding `-plugin` on the end is a bit redundant. We don't typically name deployments `kubevirt-deployment` and pods `kubevirt-pod` for example. If there is no `-plugin` suffix in the `ConsolePlugin` resource name, then it wouldn't be in the JSON filename or i18n namespace.